src/clients/customerCards/create/types.ts

```diff
@@ -0,0 +1,18 @@
+import type { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+import type { Options } from '@src/types';
+
+export declare type CustomerCardsCreateClient = {
```

Comment:
como deve der? plural ou singular?
CustomerCards ou CustomerCard

Reply:
I looked at other SDKs and it is all in singular, so it will be in the singular too